charts/argo-workflows/README.md

```diff
@@ -211,6 +211,7 @@ Fields to note:
 | controller.serviceAnnotations | object | `{}` | Annotations to be applied to the controller Service |
 | controller.serviceLabels | object | `{}` | Optional labels to add to the controller Service |
 | controller.serviceMonitor.additionalLabels | object | `{}` | Prometheus ServiceMonitor labels |
+| controller.serviceMonitor.apiVersion | string | `"monitoring.coreos.com/v1"` | ApiVersion for the serviceMonitor resource |
```

Comment:
Maybe it would be better to use the structure we're already using at the argo-cd chart:
https://github.com/argoproj/argo-helm/blob/508162f1cc9852ccfbac6fb46b8e7c180d95d298/charts/argo-cd/values.yaml#L14-L16

Reply:
I cannot tell if GKE api version is changing much frequently.
Having a default value for Prometheus might be convenient. In any case, I've left an extra check to prevent the apiVersion is empty when the service monitor is enabled.

@mkilchhofer: Do I still need to proceed with the suggested change?

Reply:
You're right. The capability validation won't pass if `apiVersion` is intentionally set to empty.

Remove the `required` validation in 736df43

Reply:
I only wanted to challenge where to put the variable, not the details about the feature to override.

So I was talking about:
`.Values.apiVersionOverrides.xyz`
vs.
`.Values.controller.serviceMonitor.apiVersion`